tools/db_stress.cc

```diff
@@ -198,6 +205,10 @@ DEFINE_int32(max_background_compactions,
              "The maximum number of concurrent background compactions "
              "that can occur in parallel.");
 
+DEFINE_int32(num_bottom_pri_threads, 0,
```

Comment:
Never mind. Realized that it should be 0 for the bottom pri thread pool to be disabled by default.